### ISSUE TITLE
Condition event player

### ISSUE THREAD

**Issue by jabdoa2** [missionpinball/mpf#1022]
Will only execute the first matching condition and work like event_player otherwise

**Comment by avanwinkle** [comment]
I've begun work on my fork that implements something to this effect. It doesn't handle "first matching" but rather "all matching", yet it allows the use of conditions on the _handler_ of an event instead of just the _trigger_. For example:

```
reenable_nonrecruit_modes:
    - start_mode_shadowbroker_base
    - start_mode_shopping
    # Prior to collector attack: overlord; after: arrival
    - start_mode_overlordlight{device.achievements.collectorship.state!="complete"}
    - start_mode_arrival{device.achievements.collectorship.state=="complete"}
```

My update also includes conditions for asset pools (e.g. `sound_pools`) that evaluate the inclusion of assets during random/sequential selection:
```
sound_pools:
  randomsquad_recruitcomplete:
    type: random
    track: voice
    sounds:
      - missioncomplete_voice_miranda_1
      - missioncomplete_voice_jack_1
      - missioncomplete_voice_garrus_1{current_player.status_garrus<=1}
      - missioncomplete_voice_jack_1{current_player.status_jack<=1}
      - missioncomplete_voice_grunt_1{current_player.status_grunt<=1}
      - missioncomplete_voice_mordin_1{current_player.status_mordin<=1}
      - missioncomplete_voice_samara_1{current_player.status_samara<=1}
 ```

I still have to write tests and gracefully handle some errors, but I'll put up a PR in the next few days!

**Comment by jabdoa2** [comment]
Looks awesome

**Comment by avanwinkle** [comment]
PR for conditional event handlers: https://github.com/missionpinball/mpf/pull/1067

😄

**Comment by jabdoa2** [comment]
fixed in #1067
